configs/indexes.json

```diff
@@ -156,7 +156,8 @@
     "DEFITVL": [
         {
             "URL": "https://api.defillama.com/charts",
-            "jsonPath": "$[-1:].totalLiquidityUSD"
+            "jsonPath": "$[-1:].totalLiquidityUSD",
+            "interval":600
```

Comment:
I think defilama doesn't need an interval. Need to add the defipulse here with the interval.

```
{
	"URL": "https://data-api.defipulse.com/api/v1/defipulse/api/MarketData",
    "jsonPath": "$.All.total",
	"interval":600
   }
```